exchanges/gateio/gateio_wrapper.go

```diff
@@ -210,6 +210,8 @@ func (g *Gateio) Setup(exch *config.Exchange) error {
 		GenerateSubscriptions:  g.GenerateDefaultSubscriptions,
 		ConnectionMonitorDelay: exch.ConnectionMonitorDelay,
 		Features:               &g.Features.Supports.WebsocketCapabilities,
+		FillsFeed:              g.Features.Enabled.FillsFeed,
+		TradeFeed:              g.Features.Enabled.TradeFeed,
```

Comment:
Not being generated in subs if trade saving is false. There is also no subscriptions it seems for auth websocket but out of scope. 

```diff
diff --git a/exchanges/gateio/gateio_websocket.go b/exchanges/gateio/gateio_websocket.go
index c15c21645..2a339b657 100644
--- a/exchanges/gateio/gateio_websocket.go
+++ b/exchanges/gateio/gateio_websocket.go
@@ -668,7 +668,7 @@ func (g *Gateio) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, e
                                params["interval"] = kline.HundredMilliseconds
                        }
                        if spotTradesChannel == channelsToSubscribe[i] {
-                               if !g.IsSaveTradeDataEnabled() {
+                               if !g.IsTradeFeedEnabled() {
                                        continue
                                }
                        }
```